README.md

```diff
@@ -228,6 +245,8 @@ To remove the library from your system use the provided uninstall target:
 make uninstall-kahypar
 ```
 
+Note, our library interfaces uses Mt-KaHyPar Fast. We are currently working on a solution to also integrate Mt-KaHyPar Strong.
```

Comment:
why not two library targets? also we might want to let the binaries just use the corresponding library --> compile only once. I think KaHiP does this.